### DISCUSSION TITLE
slash-create MessageOptions not compatible with discord.js MessageOptions

### DISCUSSION THREAD

**Discussion by tescher** [Snazzah/slash-create#363] [Q&A] [answered]
See below. Using discord.js 13.6.0. Do I need to upgrade something?

![image](https://user-images.githubusercontent.com/2557404/187691142-a18a0c6d-4fd4-4a54-becb-cb6457726319.png)

**Comment by sudojunior** [comment, marked as answer]
Third-party libraries _like `discord.js`_ are not directly supported. [`<Context>.send`](https://slash-create.js.org/#/docs/main/master/class/MessageInteractionContext?scrollTo=send) must be provided either a content string or [MessageOptions](https://slash-create.js.org/#/docs/main/master/typedef/MessageOptions).

Related to #275
